validator/sawtooth_validator/server/cli.py

```diff
@@ -47,27 +47,56 @@ def parse_args(args):
     return parser.parse_args(args)
 
 
+def ensure_directory(sawtooth_home_path, fallback_path):
```

Comment:
This assumes a single default path, which isn't accurate as the windows default will differ from the Linux default.